packages/components/src/form-token-field/suggestions-list.tsx

```diff
@@ -157,7 +158,7 @@ export function SuggestionsList<
 				);
 				/* eslint-enable jsx-a11y/click-events-have-key-events */
 			} ) }
-			{ suggestions.length === 0 && (
+			{ suggestions.length === 0 && ! isLoading && (
```

Comment:
The `SuggestionsList` component is shared, which is why we're passing the `isLoading` prop, correct?

Reply:
right